docs/organizations/managing-security-and-risk.md

```diff
@@ -32,29 +32,85 @@ Each of these areas displays a count of matching items and a **Review** button t
 
 The Security and Risk Management item list displays a list of items, filtered by status and sorted by due date ascending.
 
-To access the item list, access the [dashboard](#dashboard) and click the **Review** button under the area of interest, based on the desired filtering.
+To access the item list, access the [dashboard](#dashboard) and click the **Review** button under an area of interest, based on the desired filtering.
 
-To find out more about an item, click its **Details** column. This redirects you to a filtered view of the **Repository issues** page, showing only the item of interest.
+To open the originating issue of an item, click the **Details** column of the item of interest.
 
 ![Security and Risk Management item list](images/security-risk-management-item-list.png)
 
+## Managing integrations
+
+Integrations let you detect security vulnerabilities across a number of platforms and display them all as security items on the unified item list.
+
+The following section explains how to integrate with each of the supported platforms.
+
+=== "Codacy"
+
+    The Codacy integration is active by default and can't be deactivated.
+
+=== "Jira Cloud"
+
+    To install the Jira Cloud integration:
+
+    1.  Open your organization **Security and Risk** page and click the **Configure** button. This opens the configuration page.
+
+    1.  On the right-hand side of the page, under the Integrations list, find the Jira entry and click the **Install** button. This redirects you to Atlassian's website.
+
+        ![Security and Risk Management Jira integration](images/security-risk-management-integration-jira.png)
+
+    1.  On Atlassian's website, authorize Codacy to access your Atlassian account. Once successful, you're redirected back to Codacy.
+
+        !!! note
+            Use a Jira account with admin permissions when authorizing Codacy. This lets Codacy access all issues.
+
+        Upon installation, Codacy automatically imports all Jira tickets that were created within the 90-day period preceding the integration and tagged with the label "security" and displays them on the item list, along with items from other sources.
+        
+        Jira-related items are then synchronized daily with the matching Jira issues throughout their lifecycle.
+
+    To uninstall the Jira integration:
+
+    1.  Open your organization **Security and Risk** page and click the button **Configure** to open the configuration page.
+
+    1.  Find the Jira entry under the Integrations area and click the **Options** button (identified by three dots), then select **Uninstall** and confirm.
```

Comment:
Here we say "Integrations area", but above we have an "Integrations list".
Let's decide which fits best and keep consistency.